webui/react/src/types.ts

```diff
@@ -710,6 +710,7 @@ export interface TrialItem extends StartEndTimes {
   logRetentionDays?: number;
   taskId?: string;
   metadata?: JsonObject;
+  logSignal?: string;
```

Comment:
do we expected `logSignal` to be undefined? in proto files, its not optional. is it intentional? 
Does that make sense to add `logSignal` in the required fields in proto?

Reply:
I made `logSignal` optional because `TrialItem` might be use somewhere else without `Trialv1Trial`

Reply:
is it a good idea to add `optional` keyword in proto then?